packages/sdk/src/components/grid/utils/utils.ts

```diff
@@ -23,9 +25,23 @@ export const requestTimeout = (callback: () => void, delay: number): ITimeoutID
   return timeoutID;
 };
 
-export const isWindowsOS = () => {
-  const agent = navigator.userAgent.toLowerCase();
-  return /win32|wow32|win64|wow64/.test(agent);
+export const getWheelDelta = (event: WheelEvent, coordInstance?: CoordinateManager) => {
```

Comment:
It seems that this should not be bound to coordinate, and the preset page height and line height should be passed in directly.
```
const getWheelDelta = ({ 
  event, 
  pageHeight, 
  lineHeight
}: {
  event: WheelEvent, 
  pageHeight?: number, 
  lineHeight?: number
}) => {
  //...
}
```